src/platform/silabs/KeyValueStoreManagerImpl.cpp

```diff
@@ -299,6 +317,36 @@ void KeyValueStoreManagerImpl::KvsMapMigration(void)
     }
 }
 
+void KeyValueStoreManagerImpl::KvsKeyMapCleanup(void * argument)
```

Comment:
Would add a comment specifying that this thread only needs to be run once per power cycle.

FreeRTOS api `vTaskDelete` is quite explicit, CMSIS `osThreadExit` a little less...

Reply:
done